script/pptemplate

```diff
@@ -1,4 +1,4 @@
-#!perl -w
+#!/usr/bin/env perl
```

Comment:
This change is incorrect. EUMM updates it to be exactly the same Perl as the one installing it. Please leave it as is.

Reply:
Ok, I'll revert that!